### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by rogerkibbe**
Misc cleanup

Deleted

- quiz-questions-left-statement - not used redundant with quiz-questions-left-fragment
- advance-quiz-question) - not used
- quiz-total-questions-statement - not used

Changed:

- Answer_Input_View - used all the fragments and added comment on the fragment usage
- ScoreBook_Result.dialog - added correct answer fragment
- Quiz_Input_View - fixed deprecation
- capsule.bxb - fixed deprecation

Added:

- Quiz_Show_Correct.dialog - tells the user the correct answer if they answered incorrectly
- Training to use the quiz name when asked which quiz

Moved:
All of the dialog and views to resources/en - most of these files had some English in them, but not all. I think it would be confusing though to mix and match and easier to just have all of the views and dialogs together.

### CODE REVIEW

File: resources/en/view/Answer_Input.view.bxb [hunk 1]
```diff
@@ -3,10 +3,14 @@ input-view {
...
+  // Mix and match the Macros to get your desired dialog. Macros include:
+  // score-statement: Says correct or incorrect
+  // quiz-show-correct: Tells the correct answer if the user answered incorrectly
+  // quiz-questions-left-fragment: How many questions are left
   message {
     if (exists(action.state.lastScore)) {
-      template ("#{macro('score-statement', action.state.lastScore)} #{value(action.state.currentQuestion.textToDisplay)}") {
-        speech ("#{macro('score-statement', action.state.lastScore)}... #{value(action.state.currentQuestion.textToSpeak)}")
+      template ("#{macro('score-statement', action.state.lastScore)} #{macro('quiz-show-correct', action.state.lastScore)} #{macro('quiz-questions-left-fragment', action.state.scoreBook.questionsLeftCount)} #{value(action.state.currentQuestion.textToDisplay)}") {
+        speech ("#{macro('score-statement', action.state.lastScore)} #{macro('quiz-show-correct', action.state.lastScore)} #{macro('quiz-questions-left-fragment', action.state.scoreBook.questionsLeftCount)}... #{value(action.state.currentQuestion.textToSpeak)}")
```
Comment (siamakh) [inline resources/en/view/Answer_Input.view.bxb:13]: Make sure TTS reads the speech channel okay when statements are concatenated. I found out that TTS is not pausing enough between the statements unless I add "..." between them.

File: resources/en/view/Quiz_Input.view.bxb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+input-view {
```
Comment (siamakh) [inline resources/en/view/Quiz_Input.view.bxb:1]: We usually keep the views in base (and make sure it is using template macros for dialogs that are in target language folders). This makes it easier to add new device and locale targets.

File: resources/base/dialog/ScoreBook_Result.dialog.bxb [hunk 3]
```diff
@@ -1,11 +0,0 @@
-dialog (Result) {
```
Comment (siamakh) [inline resources/base/dialog/ScoreBook_Result.dialog.bxb:1]: I think as long as the dialog is made of statements that are macros themselves they can still stay in base, but not sure, might be confusing to devs so maybe okay to move them all to target langs.

File: resources/base/dialog/macros/AdvanceQuiz_Question.dialog.bxb [hunk 4]
```diff
@@ -1,19 +0,0 @@
-template-macro-def (advance-quiz-question) {
```
Comment (siamakh) [inline resources/base/dialog/macros/AdvanceQuiz_Question.dialog.bxb:1]: Thanks for moving this.

**Review by siamakh — COMMENTED** [review]
Looks good. Couple of minor comments.

**Comment by rogerkibbe** [comment]
Great comments - going to resubmit a PR with the above suggestions

**Pull request closed without merging**